File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        res=[0]*(len(nums)*2)
        n=len(nums)
        for i in range(n):
            res[i]=nums[i]
            res[n+i]=nums[i]
        return res

File: Data Structures & Algorithms/daily-temperatures/submission-0.java
class Solution {
    public int[] dailyTemperatures(int[] temperatures) {
        int n = temperatures.length;
        int[] res = new int[n];

        for(int i = 0; i < n; i++) {
            for(int j = i+1; j < n; j++) {
                if(temperatures[j] > temperatures[i]) {
                    res[i] = j-i;
                    break;
                }
            }
        }

        return res;
    }
}
/*
return an array of size n where n is size of temperatures
result[i] = days after i BEFORE a warmer temp appears on a future day
if NO days in future for warmer temperature for ith day, set to 0

temperatures=[30,38,30]

result=[1,0,0]
we know last day will be 0

STACK is LIFO

populate stack:
    iterate through temperatures:
        - put temp[i] in stack
iterate through temperatures:

*/
